src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -1651,10 +1651,21 @@ static void uct_ib_mlx5_devx_check_odp(uct_ib_mlx5_md_t *md,
         goto no_odp;
     }
 
-    ucs_debug("%s: ODP is supported, version %d",
-              uct_ib_device_name(&md->super.dev), version);
-    md->super.reg_nonblock_mem_types = UCS_BIT(UCS_MEMORY_TYPE_HOST);
     md->super.gva_mem_types          = UCS_BIT(UCS_MEMORY_TYPE_HOST);
+    md->super.reg_nonblock_mem_types = UCS_BIT(UCS_MEMORY_TYPE_HOST);
```

Comment:
shall we also update `gva_mem_types`? @Artemy-Mellanox

Reply:
added, also added gtest to check ucp_mem_map() allocates from those supported mem_types